ci/scripts/run_tests.sh

```diff
@@ -16,5 +16,4 @@ then
   EXTRA_OPTIONS="$EXTRA_OPTIONS --benchmark"
 fi
 
-python -m pytest -n 10 --dist loadscope $EXTRA_OPTIONS "$@"
-
+python -m pytest -n 8 --dist loadscope $EXTRA_OPTIONS $@
```

Comment:
Any particular theory behind this change? Were we hitting rate limits in AWS?

Reply:
The previous limit was never reached.
benchmarks -> 6 modules
runtime -> 4 modules
stability -> 4 modules

I didn't want to suddenly stress AWS even more than what we're doing now.

Reply:
Right, though it will be reached with this change. I have no objection to this, just wondering if you ran into troubles with 10 here.